tests/test_sphericaldf.py

```diff
@@ -1722,6 +1722,175 @@ def test_constantbeta_differentpotentials_dens_directint():
                              bins=11)
     return None
 
+################# INTERPOLATED POTENTIALS IN DFS ##############################
+
+# Eddington DFs with interpolated potentials
+def test_eddington_interpolatedpotentials_dens_directint():
+    # Some potentials
+    pots= [potential.HernquistPotential(amp=1.3,a=0.8),
+           potential.PlummerPotential(amp=2.3,b=1.3),
+           potential.NFWPotential(amp=1.3,a=1.5),
```

Comment:
```suggestion
```
While the tests are good and comprehensive, they seem to altogether add about 15 minutes to the test suite of the spherical DFs, almost doubling its time (and it's already one of the longest-running parts of the tests). I'm not sure why they take so long to run on GitHub actions, locally they run in 3 min. for me! But I'm going to be taking the knife to the tests to reduce the time for the tests..